common/libs/checksum/checksum.cpp

```diff
@@ -13,9 +13,8 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
-#include <stdint.h>
-
 #include "checksum.h"
+#include <stdint.h>
```

Comment:
What is going on here?  This is not clang-format's style I think?

Reply:
It is clang-format's style - confirmed by changing it back and manually running clang-format.